### ISSUE TITLE
error signaled after adding assignment, but assignment still added

### ISSUE THREAD

**Issue by joerromano** [jbclements/admiral-edu-server#34]
In the 'author' page creating a new assignment, I created a new one with the following description

```
name: Flags
id: flags
description: Supplemental homework 1.
steps:
  - id: final-submission
    instructions: Submit your final code here.
```

After clicking "Validate and Submit", I was given an alert box indicating there was an error.  However, when clicking the button again, I was given a different alert box saying the assignment already exists.

After navigating to the main page, the assignment did indeed exist and was correct.

**Comment by jbclements** [comment]
looks like the issue here is that the configuration has the wrong bucket name. Fixing this should be almost trivial, but I want to add a startup check so that the instance won't start up at all in this case.

**Comment by jbclements** [comment]
okay, added check, tested that it works.

fixed in 292576070456fe
